### PR TITLE
Update boot-passthrough.sh to be more inline with OpenCore-Boot.sh. U…

### PR DESCRIPTION
**Pull request by dcemmerson**
…pdated file references.

boot-passthrough.sh was not working for me (due to an incorrect file reference or two, probably from previous version) and some of the device settings. boot-passthrough.sh now working, so hopefully this saves someone else from having to troubleshoot and tweak.

### CODE REVIEW

File: boot-passthrough.sh [hunk 1]
```diff
@@ -10,25 +10,26 @@ OVMF_DIR="."
...
+  -device ich9-intel-hda -device hda-duplex
+  -device ich9-ahci,id=sata
+  -drive id=MacHDD,if=none,file="$REPO_PATH/mac_hdd_ng.img",format=qcow2
+  -device ide-hd,bus=sata.2,drive=MacHDD
   -drive id=OpenCoreBoot,if=none,snapshot=on,format=qcow2,file="$REPO_PATH/OpenCore-Catalina/OpenCore-Passthrough.qcow2"
   -device ide-hd,bus=sata.3,drive=OpenCoreBoot
   -netdev tap,id=net0,ifname=tap0,script=no,downscript=no -device vmxnet3,netdev=net0,id=net0,mac=52:54:00:c9:18:27
   -monitor stdio
-  -display none
+  -vga vmware
```
Comment (kholia) [inline boot-passthrough.sh:32]: This file is meant for GPU (+ HW) passthrough setups.

**Comment by kholia** [comment]
Thanks!

This file is meant for GPU (+ HW) passthrough setups. I will pull in some of the changes manually while keeping some of the sample `passthrough` lines as they are.

**Comment by dcemmerson** [comment]
Oh ok, awesome! 

I guess I wasn't very clear in my original comments, I was using the boot-passthrough.sh script just to pass through the USB controller and got a bit of tunnel vision in the process...

Thanks so much! This tool works great!

**Comment by kholia** [comment]
Thank you!

Improving the documentation around this area is something we would like to work on.

https://github.com/kholia/OSX-KVM#setting-expectations-right

**Comment by kholia** [comment]
I have pushed the following patch to the repository.

```diff
diff --git a/boot-passthrough.sh b/boot-passthrough.sh
index 604fa32..85630dc 100755
--- a/boot-passthrough.sh
+++ b/boot-passthrough.sh
@@ -1,5 +1,23 @@
 #!/usr/bin/env bash
 
+# Special thanks to:
+# https://github.com/Leoyzen/KVM-Opencore
+# https://github.com/thenickdude/KVM-Opencore/
+# https://github.com/qemu/qemu/blob/master/docs/usb2.txt
+#
+# qemu-img create -f qcow2 mac_hdd_ng.img 128G
+#
+# echo 1 > /sys/module/kvm/parameters/ignore_msrs (this is required)
+
+############################################################################
+# NOTE: Tweak the "MY_OPTIONS" line in case you are having booting problems!
+############################################################################
+
+MY_OPTIONS="+pcid,+ssse3,+sse4.2,+popcnt,+avx,+aes,+xsave,+xsaveopt,check"
+
+# This script works for Big Sur, Catalina, Mojave, and High Sierra. Tested with
+# macOS 10.15.6, macOS 10.14.6, and macOS 10.13.6
+
 ALLOCATED_RAM="3072" # MiB
 CPU_SOCKETS="1"
 CPU_CORES="2"
@@ -8,6 +26,13 @@ CPU_THREADS="4"
 REPO_PATH="./"
 OVMF_DIR="."
 
+# Note: This script assumes that you are doing CPU + GPU passthrough. This
+# script will need to be modified for your specific needs!
+#
+# We recommend doing the initial macOS installation without using passthrough
+# stuff. In other words, don't use this script for the initial macOS
+# installation.
+
 # shellcheck disable=SC2054
 args=(
   -enable-kvm -m "$ALLOCATED_RAM" -cpu host,kvm=on,vendor=GenuineIntel,+invtsc,vmware-cpuid-freq=on
@@ -19,10 +44,10 @@ args=(
   -device vfio-pci,host=01:00.1,bus=port.1
   -usb -device usb-kbd -device usb-tablet
   -device isa-applesmc,osk="ourhardworkbythesewordsguardedpleasedontsteal(c)AppleComputerInc"
-  -drive if=pflash,format=raw,readonly,file=OVMF_CODE.fd
-  -drive if=pflash,format=raw,file=OVMF_VARS.fd
+  -drive if=pflash,format=raw,readonly,file="$REPO_PATH/$OVMF_DIR/OVMF_CODE.fd"
+  -drive if=pflash,format=raw,file="$REPO_PATH/$OVMF_DIR/OVMF_VARS-1024x768.fd"
   -smbios type=2
-  -drive id=MacHDD,if=none,file=./mac_hdd.img
+  -drive id=MacHDD,if=none,file=./mac_hdd_ng.img
   -device ide-drive,bus=sata.2,drive=MacHDD
   -drive id=OpenCoreBoot,if=none,snapshot=on,format=qcow2,file="$REPO_PATH/OpenCore-Catalina/OpenCore-Passthrough.qcow2"
   -device ide-hd,bus=sata.3,drive=OpenCoreBoot
```

**Pull request closed without merging**